Changes

```diff
@@ -1,5 +1,29 @@
 Release history for Zonemaster component Zonemaster-Backend
 
+v6.1.0 2020-11-06 (public release version)
+
+ [Features]
+ - Adds full support of a light weight SQLite database backend
+   as an alternative to MariaDB or PostgreSQL database daemons
+   (#695, #690, #177
+ - Extends zmtest and zmb with new functions (#675)
```

Comment:
Maybe say something about the character of the new functionality?

Reply:
Updated description.